Week3/Solution/week3_solution.txt

```diff
@@ -0,0 +1,261 @@
+Write a python script to perform CRUD operation on following table of "ESM.db" database.
+
+Employee ( eid, ename, dept, basic, branch). eid must be primary key.
+
+Department : Account, Inventory, IT, HR.
+
+Peform Following Operation.
+
+1. Create table
+
+->
+from sqlite3 import *
+
+conn = connect('C:\\sqlite\\ESM.db')

```

Comment:
why you use '\\' ?

Reply:
'\' blackslah is used for line continuation in python
if we don't want to use '\' we can also used """ <statment> """